src/pages/get-eth.tsx

```diff
@@ -182,309 +150,243 @@ const GetEthPage = ({
-            {t("page-get-eth-your-address-wallet-link")}
-          </InlineLink>
-          <OldHeading
-            as="h3"
-            fontSize={{ base: "xl", md: "2xl" }}
-            lineHeight={1.4}
-          >
-            {t("page-get-eth-your-address")}
-          </OldHeading>
-          <Text>{t("page-get-eth-your-address-desc")}</Text>
-          <Flex
-            justifyContent="space-between"
-            bg="#191919"
-            borderRadius="base"
-            p={2}
-            mb={6}
-            userSelect="none"
-            flexDir={{ base: "column-reverse", lg: "initial" }}
-          >
-            <Text fontFamily="monospace" color="white" mb={0} fontSize="xs">
-              0x0125e2478d69eXaMpLe81766fef5c120d30fb53f
-            </Text>
-            <Text
-              textTransform="uppercase"
-              fontSize="sm"
-              color="fail300"
-              mb={0}
-              mx={4}
-            >
-              {t("page-get-eth-do-not-copy")}
-            </Text>
-          </Flex>
-          <Text>{t("page-get-eth-your-address-desc-3")}</Text>
-          <OldHeading
-            as="h3"
-            fontSize={{ base: "xl", md: "2xl" }}
-            lineHeight={1.4}
-          >
-            {t("page-get-eth-wallet-instructions")}
-          </OldHeading>
-          <Text>{t("page-get-eth-wallet-instructions-lost")}</Text>
-        </RightColumn>
-      </TwoColumnContent>
-      <Divider />
-      <CalloutBanner
-        className="mx-4 mb-40 mt-24"
-        titleKey="page-get-eth:page-get-eth-use-your-eth"
-        descriptionKey="page-get-eth:page-get-eth-use-your-eth-dapps"
-        image={dapps}
-        alt={t("page-index:page-index-sections-individuals-image-alt")}
-        imageWidth={600}
-      >
-        <Box>
-          <ButtonLink href="/dapps/">
-            {t("page-get-eth-checkout-dapps-btn")}
-          </ButtonLink>
-        </Box>
-      </CalloutBanner>
-      <FeedbackCard />
-    </Page>
+          <CentralizedExchanges lastDataUpdateDate={lastDataUpdateDate} />
+        </div>
+
+        <Stack className="gap-12">
+          <h2 id="dex" className="text-2xl leading-6 md:text-3xl">
+            {t("page-get-eth-dexs")}
+          </h2>
+          <TwoColumnContent>
+            <Stack className="gap-4">
+              <h3 className="text-xl leading-6 md:text-2xl">
+                {t("page-get-eth-what-are-DEX's")}
+              </h3>
+              <p>{t("page-get-eth-dexs-desc")}</p>
+              <p>
+                {t("page-get-eth-dexs-desc-2")}{" "}
+                <InlineLink href="/smart-contracts">
+                  {t("page-get-eth-smart-contract-link")}
+                </InlineLink>
+              </p>
+              <p>{t("page-get-eth-dexs-desc-3")}</p>
+              <p>{t("page-get-eth-need-wallet")}</p>
+              <ButtonLink href="/wallets/find-wallet/">
+                {t("page-get-eth-get-wallet-btn")}
+              </ButtonLink>
```

Comment:
<img width="805" alt="image" src="https://github.com/user-attachments/assets/3ba63b9d-82cf-4600-884b-8b3baa106715">

may need a `w-fit` on this if we want it to match production

<img width="820" alt="image" src="https://github.com/user-attachments/assets/eb66a4e0-8e34-4749-8a00-48e499adef25">

```suggestion
              <ButtonLink href="/wallets/find-wallet/" className="w-fit">
                {t("page-get-eth-get-wallet-btn")}
              </ButtonLink>
```